lib/openshift_client.rb

```diff
@@ -52,5 +52,16 @@ def initialize(uri,
     def all_entities
       retrieve_all_entities(ENTITY_TYPES)
     end
+
+    def process_template(template)
```

Comment:
users would need some documentation/example how to do it (without searching the tests)